index.html

```diff
@@ -24,7 +24,7 @@
     <div class="bigwords">
       <span class="s">Veneur</span>
 
-      is a distributed server implementation of the statsd protocol.
+      is a distributed, fault-tolerant pipeline for runtime data.
```

Comment:
Hm, is that right? How's "the basis for your distributed, fault-tolerant observability pipeline"? (: